panel/widgets/indicators.py

```diff
@@ -1137,6 +1138,9 @@ class Tqdm(Indicator):
     layout = param.ClassSelector(class_=(Column, Row), precedence=-1, constant=True, doc="""
         The layout for the text and progress indicator.""",)
 
+    lock = param.ClassSelector(class_=object, default=None,
```

Comment:
@philippjfr would know much better than me if there could be any unanticipated sideeffects of using a Lock like this with Panel?

Reply:
No real problem we just have to handle the pyodide case gracefully.